README.md

```diff
@@ -21,6 +21,13 @@
 - You can help us out by resolving TODOs and replying on issues
 - Of course, all feedback, positive and negative, also really helps to improve the project
 
+### How to run the project
```

Comment:
It could be better if we can move it above "How to help" section and make it the same title style as "How to help" with different emoji